### DISCUSSION TITLE
about agent tomcat plugin and springmvc plugin

### DISCUSSION THREAD

**Discussion by JihanLin** [apache/skywalking#10792] [Q&A] [answered]
Excuse me, my link has passed through the Tomcat plugin (I logged it). Why is the tracking showing the SpringMVC component? Can there be any coverage

my pom.xml dependency

`
<dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-web</artifactId>
            <version>2.7.10</version>
 </dependency>
`

my api:

`
@RestController
public class Controller {

    @PostMapping("/testcase/body")
    public String body(@RequestBody String body) {
        System.out.println("-----controller:" + body);
        return body;
    }

}

`
the tracking showing:

![image](https://github.com/apache/skywalking/assets/12154731/cf4cd72b-6054-4cde-a7ce-c1ee1bee8261)

**Comment by JihanLin** [comment]
it not show Tomcat plugin's span info

**Comment by wu-sheng** [comment, marked as answer]
SkyWalking wouldn't. As it is not meaningful. SpringMVC plugin has overrides the Tomcat span automatically at the agent side.
